src/github/push.unit.test.ts

```diff
@@ -0,0 +1,69 @@
+import { pushWebhookHandler } from "./push";
+import { WebhookContext } from "routes/github/webhook/webhook-context";
+import { getLogger } from "config/logger";
+import { GitHubPushData, GitHubRepository } from "../interfaces/github";
+import { enqueuePush } from "../transforms/push";
+
+jest.mock("../transforms/push");
+
+const GHES_GITHUB_INSTALLATION_ID = 1234;
+const GHES_GITHUB_APP_ID = 111;
+const GHES_GITHUB_UUID = "xxx-xxx-xxx-xxx";
+const GHES_GITHUB_APP_APP_ID = 1;
+const GHES_GITHUB_APP_CLIENT_ID = "client-id";
+
+describe("PushWebhookHandler", ()=>{
+	let jiraClient: any;
+	let util: any;
+	beforeEach(() => {
+		jiraClient = { baseURL: jiraHost };
+		util = null;
+	});
+	describe("GitHub Cloud", ()=>{
+		it("should NOT push GitHubAppConfig", async ()=>{
```

Comment:
I think `should NOT push GitHubAppConfig` is a bit misleading, because no one is pushing the `GitHubAppConfig`. Maybe call it `should be called WITHOUT GitHubAppConfig` instead?

Reply:
👍 fixed.